client/js/controllers/game.js

```diff
@@ -33,23 +75,25 @@ angular.module('mean.system')
         }
       };
 
-      $scope.checkDisable = selectedEmail => $scope.invitedUsers.indexOf(selectedEmail) === -1;
+      $scope.checkDisable = selectedEmail => $scope.invitedUsers.indexOf(selectedEmail) !== -1;
 
       $scope.selectUser = (selectedEmail) => {
         if ($scope.invitedUsers.length <= 11) {
           const mailObject = {
             To: selectedEmail,
             Link: document.URL
           };
+          $scope.invitedUsers.push(selectedEmail);
           userSearch.sendInvite(mailObject)
             .then(() => {
-              $scope.invitedUsers.push(selectedEmail);
             })
             .catch(() => {
               const myModal = $('#limit_modal');
               myModal.find('.modal-body')
                 .text('Error occured while inviting users');
               myModal.modal('show');
+              // remove invited from array;
+              $scope.invitedUsers.pop(selectedEmail);
```

Comment:
cool idea, but rather than pop, you should search for the index of the email and splice
why? what if user clicked two invites and the first one fails